gwsurrogate/catalog.py

```diff
@@ -61,7 +61,7 @@
                None)
 
 _surrogate_world['SpEC_q1_10_NoSpin'] = \
-  surrogate_info('https://zenodo.org/record/3348115/files/SpEC_q1_10_NoSpin_nu5thDegPoly_exclude_2_0.h5',
```

Comment:
Why are we updating the zenodo record for each of these other models?

Reply:
its true this change isn't really needed -- the other model's h5 files are unchanged in this new record. The updated URL is just to point to the most recent record so that all Zenodo-hosted files consistently point to https://zenodo.org/records/13376190.

If desired, we could revert back this (and similar) blocks of code.